src/numbers/Main.java

```diff
@@ -0,0 +1,20 @@
+package numbers;
+
+import java.util.Scanner;
+
+public class Main{
```

Comment:
Cant understand the program structure. Can you create a class and block scheme diagrams for visibility?

Reply:
Do you mean UML class diagram?